tests/kafkatest/directory_layout/kafka_path.py

```diff
@@ -0,0 +1,113 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from kafkatest.directory_layout.path_resolver import PathResolver
+from kafkatest.version.version import TRUNK
+
+import importlib
+import os
+
+
+# TODO - describe how overriding can work
+"""This module contains tools for resolving the location of directories and scripts.
+
+
+"""
+
+DEFAULT_SCRATCH_ROOT = "/mnt"
+DEFAULT_KAFKA_INSTALL_ROOT = "/opt"
+KAFKA_PATH_RESOLVER_KEY = "kafka-path-resolver"
+DEFAULT_KAFKA_PATH_RESOLVER = "kafkatest.directory_layout.kafka_path.KafkaSystemTestPathResolver"
+
+
+JARS = {
+    "trunk": {
+        "core": ["core/build/*/*.jar"],
+        "tools": ["tools/build/libs/kafka-tools*.jar", "tools/build/dependant-libs*/*.jar"]
+    }
+}
+
+
+"""
+core jars
+
+def core_jar_paths(self, node, lib_dir_name):
```

Comment:
I know this is wip, but are these commented out chunks of code going to be used?